### ISSUE TITLE
Broken link at the bottom of telepresence.io/discussion/overview

### ISSUE THREAD

**Issue by vemv** [telepresenceio/telepresence#817]
This one:

![image](https://user-images.githubusercontent.com/1162994/47909802-42f86780-de91-11e8-90e4-be648042e233.png)

Thanks - Victor

**Comment by vemv** [comment]
Fix:

```diff
- <a href="tutorials/kubernetes-rapid">quick start that uses the VPN method</a>
+ <a href="/tutorials/kubernetes-rapid">quick start that uses the VPN method</a>
```

**Comment by ark3** [comment]
Fixed in #819
